packages/frama-clang/frama-clang.0.0.15/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "Frama-C plug-in based on Clang for parsing C++ files"
+description:
+  "This Frama-C plug-in parse C++ files that may content ACSL++ annotations."
+maintainer: "Virgile.Prevosto@cea.fr"
+authors: "Virgile Prevosto"
+license: "LGPL-2.1-only"
+tags: ["formal specification" "C++" "plugins" "ACSL" "ACSL++"]
+homepage: "https://frama-c.com/fc-plugins/frama-clang.html"
```

Comment:
Do you mind also adding a `dev-repo` field?

Reply:
Done in [1e62117](https://github.com/ocaml/opam-repository/pull/25582/commits/1e621173b0be724446cee9acfcfc570fbc790d77)